packages/core/src/Engine.ts

```
@@ -407,6 +425,69 @@
+   * Add a post process pass to the manager.
+   * @param pass - Post process pass to add
+   */
+  addPostProcessPass(pass: PostProcessPass): void {
+    if (pass.engine !== this) {
+      throw "The pass is not belong to this engine.";
+    }
+
+    const passes = this._postProcessPasses;
+    if (passes.indexOf(pass) === -1) {
+      passes.push(pass);
+      pass.isActive && (this._postProcessPassNeedRefresh = true);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)